### PR TITLE
feat: add node, disk, server, remote-command related command 

### PR DESCRIPTION
**Pull request by Shuo-Jia**
Add commend:
```shell
# list node info
>>> nodes
# disk info
>>> disk-replica 
>>> disk-capacity
# disk migrate
>>> disk-migrate
# server info
>>> server-info
# remote command
>>> remote-command

```
`command -h` see detail argument

### CODE REVIEW

File: executor/client.go [hunk 1]
```diff
@@ -26,3 +32,65 @@ func NewClient(writer io.Writer, metaAddrs []string) *Client {
...
+	}
+	return adminClient.NewRemoteCmdClient(addr, nodeType), nil
+}
+
+func (client *Client) GetPerfCounterClient(addr string) (*aggregate.PerfSession, error) {
+	err := client.validateReplicaAddress(addr)
+	if err != nil {
+		return nil, err
+	}
+	return aggregate.NewPerfSession(addr), nil
```
Comment (neverchanje) [inline executor/client.go:65]: use PerfClient instead, PerfSession is low level primitive.

File: executor/remote_command.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+	// formats into tabular
+	tabular := tablewriter.NewWriter(client)
+	tabular.SetHeader([]string{"node", "result"})
+	tabular.Append([]string{addr, resp})
+	tabular.Render()
+	return nil
+}
+
+func SendRemoteCommand(client *Client, nodeType session.NodeType, addr string, cmd string, args []string) (string, error) {
```
Comment (neverchanje) [inline executor/remote_command.go:45]: i think we do not need remote command anymore

File: cmd/list_nodes.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		Help:    "list all nodes in the cluster",
+		Flags: func(f *grumble.Flags) {
+			/*define the flags*/
+			f.Bool("r", "resolve", false, "resolve input or output address")
+			f.Bool("j", "json", false, "use JSON as the format of the output results. By default tabular format is used.")
+			f.Bool("d", "detail", false, "show detail replica count in all node")
+			f.Bool("u", "usage", false, "show the usage of every node")
+			f.Bool("q", "qps", false, "show the qps/bytes/latency of every node")
+			f.String("a", "app", "", "filter one app, for example, temp")
+			f.String("o", "out", "", "save output into file")
```
Comment (neverchanje) [inline cmd/list_nodes.go:22]: do not define flag for output

File: executor/client.go [hunk 4]
```diff
@@ -26,3 +32,65 @@ func NewClient(writer io.Writer, metaAddrs []string) *Client {
...
+}
+
+func (client *Client) validateReplicaAddress(addr string) error {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	defer cancel()
+	var (
+		resp, err = client.Meta.ListNodes(ctx, &admin.ListNodesRequest{
```
Comment (neverchanje) [inline executor/client.go:72]: Call meta list nodes every time you validate an address... this is inefficient

File: executor/client.go (hunk 1 above)
Comment (Shuo-Jia) [reply executor/client.go:65]: I know `PerfClient`, I only use one node session to query counter, but query all node and then filter one of them

File: executor/remote_command.go (hunk 2 above)
Comment (Shuo-Jia) [reply executor/remote_command.go:45]: So how to control the dynamic config? such as `only_move_primary`

File: executor/client.go (hunk 4 above)
Comment (neverchanje) [reply executor/client.go:72]: cache it in other places, before calling validateReplicaAddress, you must ensure nodes are prepared already.

This is a basic responsibility separation principle.

File: cmd/remote_command.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+	})
+
+	rootCmd.AddCommand(&grumble.Command{
+		Name:  "all",
+		Help:  "send remote command to meta and replica server",
+		Flags: initFlag,
+		Run: func(c *grumble.Context) error {
+			return executor.RemoteCommand(pegasusClient, "all", c.Flags.String("node"), c.Flags.String("command"), c.Flags.String("arguments"))
+		},
+	})
```
Comment (neverchanje) [inline cmd/remote_command.go:41]: Basically no use case for `remote-command all`, remove this subcommand.

**Review by neverchanje — APPROVED** [review]

**Pull request merged**